interfaces/cython/cantera/test/test_thermo.py

```diff
@@ -1127,6 +1126,52 @@ def test_coverages_dict(self):
         self.assertNear(C[self.interface.species_index('c6*M')], 0.75)
 
 
+class TestPlasmaPhase(utilities.CanteraTest):
+    def setUp(self):
+        self.phase = ct.Solution('oxygen-plasma.yaml',
+                                 'isotropic-electron-energy-plasma',
+                                 transport_model=None)
+
+    def test_converting_electron_energy_to_temperature(self):
+        self.phase.mean_electron_energy = 1.0
+        Te = 2.0 / 3.0 * ct.electron_charge / ct.boltzmann
+        self.assertNear(self.phase.Te, Te)
+
+    def test_converting_electron_temperature_to_energy(self):
+        self.phase.Te = 10000
+        energy = self.phase.Te * 3.0 / 2.0 / ct.electron_charge * ct.boltzmann
+        self.assertNear(self.phase.mean_electron_energy, energy)
+
+    def test_set_get_electron_energy_levels(self):
+        levels = np.linspace(0.01, 10, num=9)
+        self.phase.electron_energy_levels = levels
+        self.assertArrayNear(levels, self.phase.electron_energy_levels)
+
+    def test_isotropic_velocity_electron_energy_distribution(self):
+        levels = np.linspace(0.01, 10, num=9)
+        self.phase.electron_energy_levels = levels
+        self.phase.Te = 2e5
+        mean_electron_energy = 3.0 / 2.0 * (self.phase.Te * ct.gas_constant /
+                               (ct.avogadro * ct.electron_charge))
+        self.assertNear(mean_electron_energy, self.phase.mean_electron_energy)
+
+    def test_discretized_electron_energy_distribution(self):
+        levels = np.array([0.0, 1.0, 10.0])
+        dist = np.array([0.0, 0.9, 0.01])
+        self.phase.set_electron_energy_distribution(levels, dist)
+        self.assertArrayNear(levels, self.phase.electron_energy_levels)
+        self.assertArrayNear(dist, self.phase.electron_energy_distribution)
+        self.assertNear(self.phase.mean_electron_energy, 57.551453415, 1e-4)
```

Comment:
Can you explain where this value comes from. If it's just a regression test, you can write that.